Documentation/ceph-upgrade.md

```diff
@@ -39,6 +40,16 @@ those releases.
 * [Upgrade 0.6 to 0.7](https://rook.io/docs/rook/v0.7/upgrade.html)
 * [Upgrade 0.5 to 0.6](https://rook.io/docs/rook/v0.6/upgrade.html)
 
+## Important changes in this release
+The minimum Kubernetes version has changed to v1.16. You must update to at least Kubernetes version
+v1.16 before upgrading Rook from v1.7 to v1.8.
+
```

Comment:
Even though it's also mentioned in the ceph upgrade section below, in this section how about we also mention the upgrade off Nautilus is required?

Reply:
Sounds good. 👍